functions-and-operators/sequence-functions.md

```diff
@@ -9,9 +9,145 @@ Sequence functions in TiDB are used to return or set values of sequence objects
 
 | Function name | Feature description |
 | :-------------- | :------------------------------------- |
-| `NEXTVAL()` or `NEXT VALUE FOR` | Returns the next value of a sequence |
-| `SETVAL()` | Sets the current value of a sequence |
-| `LASTVAL()` | Returns the last used value of a sequence |
+| [`NEXTVAL()`](#nextval) | Returns the next value of a sequence |
+| [`NEXT VALUE FOR`](#next-value-for) | Returns the next value of a sequence |
+| [`SETVAL()`](#setval) | Sets the current value of a sequence |
+| [`LASTVAL()`](#lastval) | Returns the last used value of a sequence |
+
+## `NEXTVAL()`
+
+The `NEXTVAL()` functions returns the next value from the sequence
+
+Example:
+
+We create a sequence called `s1`.
+
+```sql
+CREATE SEQUENCE s1;
+```
+
+```
+Query OK, 0 rows affected (0.01 sec)
+```
+
+Then we request the next value from `s1`.
+
+```sql
+SELECT NEXTVAL(s1);
+```
+
+```
++-------------+
+| NEXTVAL(s1) |
++-------------+
+|           1 |
++-------------+
+1 row in set (0.00 sec)
+```
+
+## `NEXT VALUE FOR`
+
+An alias for [`NEXTVAL()`](#nextval)
+
+```sql
+SELECT NEXTVAL(s1);
+```
+
+```
++-------------+
+| NEXTVAL(s1) |
++-------------+
+|          13 |
++-------------+
+1 row in set (0.00 sec)
+```
+
+```sql
+SELECT NEXT VALUE FOR s1;
+```
+
+```
++-------------------+
+| NEXT VALUE FOR s1 |
++-------------------+
+|                14 |
++-------------------+
+1 row in set (0.00 sec)
+```
+
+## `SETVAL()`
+
+The `SETVAL(n)` function can be use to set a sequence to a specific value.
+
+```sql
+CREATE SEQUENCE s1;
+```
+
+```
+Query OK, 0 rows affected (0.01 sec)
+```
+
+Then we request the next value from `s1`.
+
+```sql
+SELECT NEXTVAL(s1);
+```
+
+```
++-------------+
+| NEXTVAL(s1) |
++-------------+
+|           1 |
++-------------+
+1 row in set (0.00 sec)
+```
+
+Now we set the value to `10`.
+
+```sql
+SELECT SETVAL(s1, 10);
+```
+
+```
++----------------+
+| SETVAL(s1, 10) |
++----------------+
+|             10 |
++----------------+
+1 row in set (0.00 sec)
+```
+
+And now we can see that the next value (the value after `10`) is `11`.
+
+```sql
+SELECT NEXTVAL(s1);
+```
+
+```
++-------------+
+| NEXTVAL(s1) |
++-------------+
+|          11 |
++-------------+
+1 row in set (0.00 sec)
+```
+
+## `LASTVAL()`
+
+The `LASTVAL()` function returns the last value that was generated by the sequence.
```

Comment:
I think it should also be "generated by the sequence in **this session**", with 'this session' highlighted/bold text!